tests/test_base_group.py

```diff
@@ -232,3 +232,17 @@ async def test_release_authorization(self) -> None:
 
             assert authorize_csc.authorized_users == f"-{self.basegroup.get_identity()}"
             assert authorize_csc.cscs_to_change == ",".join(self.basegroup.components)
+
+    async def test_show_auth_status(self) -> None:
+        async with self.make_group(usage=Usages.DryTest):
+            await self.basegroup.show_auth_status()
+
+    async def test_assert_user_is_authorized(self) -> None:
+        async with self.make_group(
+            usage=Usages.DryTest
+        ), MockAuthorizeCsc() as authorize_csc:
+            await self.basegroup.request_authorization()
```

Comment:
Maybe add a test that `self.basegroup.assert_user_is_authorized()` actually raises an exception before `self.basegroup.request_authorization()` is called?

you can do that with:

```
with pytest.raises(AssertionError):
    self.basegroup.assert_user_is_authorized()
```